README.md

```diff
@@ -1,210 +1,440 @@
+- **port** as *interger*, optional
 
-After schema is imported, we can access tables.
-To use CREATE FOREIGN TABLE is not recommended.
+  GridDB notification port.
+
+- **database** as *string*, optional
+
+  GridDB database name. Default is `public` if skipped.
+
+- **clustername** as *boolean*, optional
+
+  GridDB cluster name.
+  
+- **notification_member** as *string*, optional
+
+  If you want to use fixed list mode instead of multicast mode, you have to specify `notification_member` instead of `host` and `port`:
+`notification_member '10.18.18.18:10001'`
+
+- **updatable** as *boolean*, optional, default `true`
+
+- **batch_size** as *integer*, optional, default `1`
+
+## CREATE USER MAPPING options
+
+`griddb_fdw` accepts the following options via the `CREATE USER MAPPING`
+command:
+
+- **username** as *string*
+
+  The griddb username to connect as.
+
+- **password** as *string*
+
+  The griddb user's password.
+
+
+## CREATE FOREIGN TABLE options
+
+`griddb_fdw` accepts the following table-level options via the
+`CREATE FOREIGN TABLE` command.
+
+- **table_name** as *string*, default name of foreign table
+
+- **updatable** as *boolean*, optional, default `true`
+
+- **use_remote_estimate** as *boolean*, optional
+
+- **batch_size** as *integer*, optional, default value comes from similar foreign server option
+
+- **fdw_startup_cost** as *interger*, optional
+
+- **fdw_tuple_cost** as *integer*, optional
+
+
+The following column-level options are available:
+
+- **column_name** as *string*, optional, default name of foreign column
 
-### update and delete
+- **rowkey** as *boolean*, optional, default `false`
 
-The container must have rowkey on [GridDB][1] in order to execute update and delete query.
 
+## IMPORT FOREIGN SCHEMA options
 
-## 3. Features
-- Support SELECT
+`griddb_fdw` supports [IMPORT FOREIGN SCHEMA](https://www.postgresql.org/docs/current/sql-importforeignschema.html)
+(when running with PostgreSQL 9.5 or later) and accepts the following custom options:
 
-For SELECT, you can enable partial execution mode as follows:
+- **recreate** as *boolean*, optional, default `false`
 
+  If 'true', table schema will be updated.
+
+
+## TRUNCATE support
+
+`griddb_fdw` yet **don't support** the foreign data wrapper `TRUNCATE` API, available
+from PostgreSQL 14.
+
+Functions
+---------
+As well as the standard `griddb_fdw_handler()` and `griddb_fdw_validator()`
+functions, `griddb_fdw` provides the following user-callable utility functions:
+
+*Functions from this FDW in PostgreSQL catalog are* **yet not described**.
+
+**griddb_get_connection()**
+
+**griddb_disconnect('server_name')**
+
+**griddb_disconnect_all()**
+
+Identifier case handling
+------------------------
+
+PostgreSQL folds identifiers to lower case by default.
+Rules and problems with griddb identifiers **yet not tested and described**.
+
+Generated columns
+-----------------
+
+Behavoiur within generated columns **yet not tested and described**. 
```

Comment:
`generated columns` has been supported and tested, here is test result: https://github.com/pgspider/griddb_fdw/blob/ae1c1632558d702ce21d659a071a32d06c240b5c/expected/15.0/griddb_fdw_post.out#L7898

So, could you add this information?

Reply:
Yes, new information have been added. Please, check.

Reply:
Thank you for supporting.